WordPress-Core/ruleset.xml

```diff
@@ -98,6 +98,16 @@
 	<!-- Covers rule: Braces should always be used, even when they are not required. -->
 	<rule ref="Generic.ControlStructures.InlineControlStructure"/>
 
+	<!-- Covers rule: parenthesis should not be used when using include, include_once, require, require_once. -->
+	<rule ref="PEAR.Files.IncludingFile.BracketsNotRequired">
+		<type>warning</type>
```

Comment:
As the rule is now moved to `Core`, this is no longer a "best practice" advise, but a hard rule, so I'd like to suggest removing the `type` change for this specific error code.

For the other two error codes, I'd leave the `type` in place as this is more contentious.
You could even consider not moving the other two error codes from this sniff and just moving the `BracketsNotRequired` one to `Core`.
In that case, you may need to add `<severity>5</severity>` to these rules in the `Extra` ruleset (as including the one code in `Core` sets all the other error codes in the sniff to severity `0` IIRC).

Also take note of the fact that WPCS explicitly does not include the other two error codes from this sniff `UseIncludeOnce` and `UseInclude`.